src/node_http_parser.cc

```diff
@@ -187,7 +188,58 @@ struct StringPtr {
   size_t size_;
 };
 
+class Parser;
+
+struct ParserComparer {
```

Comment:
nit: I think Comparator is more common than Comparer
```suggestion
struct ParserComparator {
```

Reply:
👍